### ISSUE TITLE
Gem for Deckster?

### ISSUE THREAD

**Issue by rahst12** [42sixsolutions/deckster#39]
Any chance a gem for deckster is available?  :-)

**Comment by hochardga** [comment]
Have you met Corey Herbert yet?  He could help you out with it.  We've been given a different set of requirements for it.

**Comment by rahst12** [comment]
Yea I have.. I've met with Eric W once or twice on the new way forward (pure javascript).  We (Stephanie, Elli, Andy, and I) have been doing a "clean-up" rev with the ruby on rails stuff.  Mainly, cutting the code-base down to just what we need.  In doing that, we were wondering if the gem ever came into existence.  Even if it did, we weren't totally certain if we'd cut over to it, just wondering more or less.

**Comment by hochardga** [comment]
Yeah, it was in this repo https://github.com/DecksterTeam/DecksterRails.  But due to the redirection, we no longer support that one.  The new one is https://github.com/DecksterTeam/DeckStarter.
